src/components/a-checkbox/_template.js

```diff
@@ -17,6 +17,7 @@ export default function ({
       checked="${checked}" 
       value="${value}" 
       disabled="${disabled}">
-    <span class="a-checkbox__icon"></span>${childrenFragment}
+    <span class="a-checkbox__icon"></span>
+    <div class="a-checkbox__content">${childrenFragment}</div>
```

Comment:
do we really need that extra `<div>`.
Shouldn't it be sufficient to fix the icon shrink/grow issue only.

If your need special behaviour for your use case, please consider to pass these children by yourself.
The checkbox itself isn't consider to be a multiline/multicolumn element

Reply:
agree, if not needed better be avoided

Reply:
The issue is @AndyOGo that sometimes authors add a very long title that makes it multiline so better to prevent this on code side

Reply:
I see, then it will be an issue and may better be fixed here